connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/EmbeddedKafkaCluster.java

```diff
@@ -75,7 +75,7 @@
 
     private static final Logger log = LoggerFactory.getLogger(EmbeddedKafkaCluster.class);
 
-    private static final long DEFAULT_PRODUCE_SEND_DURATION_MS = 500;
+    private static final long DEFAULT_PRODUCE_SEND_DURATION_MS = TimeUnit.MILLISECONDS.convert(120, TimeUnit.SECONDS);
```

Comment:
Wouldn't this be a bit cleaner?
```suggestion
    private static final long DEFAULT_PRODUCE_SEND_DURATION_MS = TimeUnit.SECONDS.toMillis(120); 
```